package.json

```diff
@@ -7,7 +7,7 @@
     "node": ">=10.18.1"
   },
   "scripts": {
-    "test": "node node_modules/mocha/bin/mocha test test/*.js"
+    "test": "mocha --exit"
```

Comment:
Why is `--exit` necessary here?

Reply:
When I upgraded Mocha a few months ago, tests would never finish without this flag ([`--exit`](https://mochajs.org/#-exit) was the default behavior before v4.0.0).

Since then, tests have changed and they seem to work without this option. I will double check.

Reply:
Fixed.

Reply:
verified; exits fine for me without `--exit`.